release.yaml

```diff
@@ -114,13 +114,14 @@ docker-images:
         repository: https://github.com/docker-library/postgres
         tag: 12-alpine
       metal-db:
-        name: rethinkdb
+        name: rethinkdb@sha256
         repository: https://github.com/rethinkdb/rethinkdb-dockerfiles
-        tag: 2.4.0
+        tag: 4cb806ed8ba198cb70d0a5e78f3e41f624b59c0e5d253a74a8dfa7af12a94909
```

Comment:
Without this version pin, old deployments will potentially also fail when the backup-restore-sidecar is being used. This can only happen when existing image tags are "overpushed", which was done by rethinkdb.

Reply:
Not sure if we need to backport this to older releases or not...

Reply:
I made patch releases for all prior versions to fix this. It's like v0.1.3-1, etc.